index.js

```diff
@@ -90,12 +90,16 @@ class fifos {
       if( this._allagents.has( entity.uri ) ) {
         const agent = this._allagents.get( entity.uri )
 
+        let timeout = agent.agentlag
+        if ( "NORMAL_CLEARING" == call.hangup_cause )
+          timeout = 1000
```

Comment:
Shouldn't it be:

```js
if ( "NORMAL_CLEARING" != call.hangup_cause )
```

i.e. normal clearing is the call happened - and both ends talked - therefore we need the full agent lag?